core/ir/aarch64/codec.c

```diff
@@ -1594,8 +1594,14 @@ decode_opnd_fpimm8(uint enc, int opcode, byte *pc, OUT opnd_t *opnd)
      *     = imm64:imm64 (Q=1)
      */
     union {
+#ifdef DR_HOST_AARCH64
         __fp16 f;
-        ushort i;
+        uint16_t i;
+#else
+        /* For platforms on which 16 bit (half-precision) FP not yet available */
```

Comment:
style nit: missing end punctuation `.` and missing complete sentence grammar: `is`

Reply:
Done.